### ISSUE TITLE
Tribler GUI download not working

### ISSUE THREAD

**Issue by itsdsk** [Tribler/tribler#3360]
##### Tribler version/branch+revision: #####

7 rc5

##### Operating system and version: #####

Ubuntu 16.04

##### Steps to reproduce the behavior: #####

Click download button in Tribler GUI

##### Expected behavior: #####

Torrent to be added to downloads

##### Actual behavior: #####

Most torrents aren't added to downloads, but log error if they are

##### Relevant log file output: #####

Error when torrent is added to downloads:

`ERROR   1516217978.77 LibtorrentDownloadImpl:845   Resume data failed to save: <class 'Tribler.Core.exceptions.SaveResumeDataError'>: torrent has no metadata`

Warnings when clicking download with no response:

`WARNING 1516217855.20        timeline:248   FAIL time:6 user:59249 -> permit^channel (no authorization)`

`INFO    1516217856.98   LibtorrentMgr:618   start_download: Starting in VOD mode`

`INFO    1516217856.98  LaunchManyCore:623   2834e8012619f663946ada2d49ae6a70d323bbfe.state not found`

`INFO    1516217856.98 LibtorrentDownloadImpl:276   LTMGR/DHT/session not ready, rescheduling create_engine_wrapper`

`WARNING 1516217857.85       community:500   message from {192.168.1.109:7759 108.218.140.65:7759} cannot be found in the request cache, skipping it`

`WARNING 1516217858.14       community:1895  drop a 449 byte packet duplicate message by member^global_time (2) [torrent] from <Candidate 83.161.206.62:7759>`

`INFO    1516217858.15        dispersy:1152  estimate someones WAN address is ('175.136.147.204', 53185) (WAN was ('175.136.147.204', 38433), LAN stays ('192.168.1.45', 45677))`

`INFO    1516217858.32         session:241   Error when querying http tracker: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.TimeoutError'>: User timeout caused connection failure.
] http://retracker.krs-ix.ru/announce`

`WARNING 1516217858.32 torrent_checker:233   Got session error for URL http://retracker.krs-ix.ru/announce: [Failure instance: Traceback (failure with no frames): <type 'exceptions.ValueError'>: HTTP tracker failed for url http://retracker.krs-ix.ru/announce (error: User timeout caused connection failure.)
`

**Comment by devos50** [comment]
Thank you for your report. Do I understand correctly that these torrents are not visible on the downloads page in the GUI? And could you clarify what kind of torrents you try to add (one from a channel or a .torrent file)?

**Comment by xoriole** [comment]
@itsdsk , I'm not able to reproduce the bug in Ubuntu 16.04. It could be a torrent specific issue. Is it possible to send me the actual torrent file? if yes, you can send it privately by email if you like. (See my profile for email address)

**Comment by itsdsk** [comment]
@devos50 & @xoriole thanks for your replies. Most torrents I try to download just don't show up in the GUI downloads page, and the issue is with torrents found in channels (I haven't tried downloading torrent files directly).

One torrent did start downloading after I posted yesterday, so that's working fine.

But on my system I noticed some persistence of data when updating, because I have a favourited channel that remained from rc4 to rc5, despite running `sudo apt purge tribler && autoclean && autoremove`. I will try to completely remove Tribler and torrents from my system then report back.
